slackbridge/bots.py

```diff
@@ -8,16 +9,17 @@
 
 
 class IRCBot(irc.IRCClient):
-    pass
+    def __init__(self, sc):
+        self.sc = sc
```

Comment:
Whoever originally named this `sc` did us a disservice :P

Reply:
easy enough to find and replace no?

Reply:
Haha yeah, I should have picked a better name for this variable, `sc` is not very descriptive at all, but I was getting tired of prefixing everything with `slack_` or `slack-`.